pkg/phases/argocdoperator/install.go

```diff
@@ -0,0 +1,27 @@
+package argocdoperator
+
+import (
+	"github.com/flanksource/karina/pkg/platform"
+)
+
+const (
+	Namespace = "argocd"
+)
+
+func Deploy(platform *platform.Platform) error {
+	if platform.ArgoCDOperator.IsDisabled() {
+		if err := platform.DeleteSpecs("", "argocd-operator.yaml"); err != nil {
+			platform.Warnf("failed to delete specs: %v", err)
+		}
+		return nil
+	}
+	if platform.ArgoCDOperator.Version == "" {
```

Comment:
This will never be true, a component without a version is considered disabled. Defaulting would need be to done above the IsDisabled check() or in defaults.go

Argo wouldn't be a critical component so it would be disabled by default

Reply:
The version will need to be added to the tests/cicd.yaml file to enable tests for that suite.

Reply:
> This will never be true, a component without a version is considered disabled. Defaulting would need be to done above the IsDisabled check() or in defaults.go
> 
> Argo wouldn't be a critical component so it would be disabled by default

Correct this will never be true. I just use the same pattern from `postgresloperator/install.go` 😂 . After a while I realized this but I forgot to fix. Thanks for catching this.